res/translations/source_copy_allow_list.tsv

```diff
@@ -0,0 +1,431 @@
+*	2/3 BPM
+*	3/2 BPM
+*	4/3 BPM
+*	MusicBrainz
+*	ReplayGain
+*	ReplayGain:
+*	ReplayGain 2.0 (ITU-R BS.1770)
+*	ReplayGain 1.0
+*	CUP (Cue + Play)
+*	%1 Hz
+*	iTunes
+*	Traktor
+*	Rhythmbox
+*	Serato
+*	hh:mm
+*	Opus
+*	AAC
+*	HE-AAC
+*	HE-AACv2
+*	Mono
+*	ICQ
+*	AIM
+*	IRC
+*	Soft Takeover
+*	SoftTakeover
+*	Opcode
+*	Note On
+*	Note Off
+*	CC
+*	Lancelot
+*	OpenKey
+*	Ctrl+S
+*	Ctrl+Shift+O
+*	Ctrl+P
+*	Ctrl+
+*	Ctrl+R
+*	Ctrl+L
+*	Ctrl+F
+*	Ctrl+t
+*	Ctrl+y
+*	Ctrl+u
+*	Ctrl+i
+*	Ctrl+o
+*	Ctrl+,
+*	Ctrl+q
+*	Ctrl+n
+*	Ctrl+1
+*	Ctrl+2
+*	Ctrl+3
+*	Ctrl+4
+*	Ctrl+6
+*	Ctrl+`
+*	Ctrl+Shift+L
+*	Ctrl+Shift+N
+*	Ctrl+Shift+R
+*	Ctrl+Shift+T
+*	Ctrl+Shift+E
+*	Ctrl+Shift+B
+*	Ctrl+Shift+D
+*	Ctrl+Backspace
+*	Ctrl+Space
+*	500 px
+*	250 px
+*	...
+*	 ms
+*	20 ms
+*	Pro-Logic Surround
+*	-30 LUFS
+*	-6 LUFS
+*	-12 dB
+*	0 dB
+*	+12 dB
+*	44 dB
+*	HSV
+*	RGB
+*	RGB L/R
+*	RGB Stacked
+*	VSyncTest
+*	Rot64
+*	Rot64Inv
+*	Rot64Fast
+*	Diff
+*	Spread64
+*	HercJog
+*	SelectKnob
+*	14-bit (LSB)
+*	14-bit (MSB)
+*	Bitcrusher
+*	Ping Pong
+*	Triplets
+*	Q
+*	Q 1
+*	Q 2
+*	Flanger
+*	Regen
+*	Mix
+*	Autopan
+*	Bessel4 ISO
+ca,de,el,en_CA,en_GB,es*,fr_CA,fr_CI,hu,it,nl,pt*,sl,sv	Res
+*	%1:%2
+*	Bessel8 ISO
+*	LR8 ISO
+*	BQ EQ
+*	BQ EQ/ISO
+*	Loudness
+*	16 bits
+*	24 bits
+*	Param EQ
+*	CBR
+*	Rekordbox
+*	%1: %2
+*	F12
+*	Esc
+*	Stereo
+*	2x MSAA
+*	4x MSAA
+*	8x MSAA
+*	16x MSAA
+*	 px
+*	TextLabel
+*	&OK
+*	LPF
+*	HPF
+*	Stem #%1
+*	FPS: n/a
+*	<i>FPS: %0/%1</i>
+de,hu,bs,cs,da,es_419,es_AR,es_CO,es_CR,es_EC,es_MX,es_PA,es_UY,et,fr,hr,id,it,lb,ms,nb,nl,nn,oc,pl,pt,pt_PT,ro,sk,sl,sv,vi	Album
+de,da,fr,lb,ms,nl,oc,sv	Genre
+de,it,pl	Bitrate:
+es,es_419,es_AR,es_CO,es_MX,ca,es_ES	Error:
+pt_BR,de,el,es_419,es_AR,es_CO,es_CR,es_EC,es_MX,es_PA,es_UY,id,it,nl,ro	Deck %1
+hu,da,de,el,es_419,es_AR,es_CO,es_CR,es_EC,es_MX,es_PA,es_UY,eu,id,nb,nl,pl,pt*,ro,sn,tr,vi	Sampler %1
+hu,id,nl,sn	Auxiliary %1
+*	Crossfader
+*	Cue
+hu,ca,de,et,fr,nb,pl,ro,sv,tr	Port
+hu,cs,de,et,fi,fr,nl,pl,pt_BR,ro,ru,sl,vi,zh*	A
+hu,pt,pt_PT,et,fi,nl,pl,pt_BR,ro,zh*	Bb
+hu,et,fi,fr,nl,pl,pt_BR,ro,ru,sl,vi,zh*	B
+hu,cs,de,et,fi,fr,nl,pl,pt_BR,ro,ru,sl,vi,zh*	C
+hu,fi,nl,pl,pt_BR,ro,sl,zh*	Db
+hu,pt,pt_PT,cs,de,et,fi,fr,nl,pl,pt_BR,ro,ru,sl,vi,zh*	D
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh*	Eb
+hu,cs,de,et,fi,fr,nl,pl,pt_BR,ro,ru,sl,vi,zh*	E
+hu,cs,de,et,fi,fr,nl,pl,pt_BR,ro,ru,sl,vi,zh*	F
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh_CN	F#
+hu,cs,de,et,fi,fr,nl,pl,pt_BR,ro,ru,sl,vi,zh*	G
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh*	Ab
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh,zh_CN,zh_HK	Am
+hu,et,fi,nl,pl,pt_BR,ro,zh*	Bbm
+hu,et,fi,nl,pl,pt_BR,ro,zh*	Bm
+hu,et,fi,nl,pl,pt_BR,ro,sl,vi,zh,zh_CN,zh_HK	Cm
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh_CN	C#m
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh*	Dm
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh,zh_CN,zh_HK	Ebm
+hu,de,et,fi,nl,pl,pt_BR,ro,sl,vi,zh*	Em
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh_CN	Fm
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh,zh_CN,zh_HK	F#m
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh,zh_CN,zh_HK	Gm
+hu,et,fi,nl,pl,pt_BR,ro,sl,zh,zh_CN,zh_HK	G#m
```

Comment:
Is this really translatable in any language? I though this was universal.

Reply:
I have filed a bug: 
https://github.com/mixxxdj/mixxx/issues/13954